src/Categories/Categories.tsx

```diff
@@ -1,12 +1,9 @@
 import React from 'react'
 import CategoryCard from '../CategoryCard/CategoryCard'
+import { CategoriesProps } from '../interface'
```

Comment:
Brilliant use of the interface file. I also love that it helped you reuse the interfaces.